skins/laika/src/store/payment/getters.ts

```diff
@@ -9,6 +9,19 @@ export const getters: GetterTree<Payment, any> = {
 	typeIsValid: function ( state: Payment ): boolean {
 		return state.validity.type !== Validity.INVALID;
 	},
+	amountValidity: function ( state: Payment ): AmountValidity {
+		/* TODO reuse configuration amounts from config files:
+		    "donation-minimum-amount": 1,
+		    "donation-maximum-amount": 100000,
+		    */
```

Comment:
I've created the ticket you can reference here: https://phabricator.wikimedia.org/T239349